demos/demo-minimal-js/index.js

```diff
@@ -75,6 +77,32 @@ const cti = new CallingExtensions({
       state.engagementId = engagementId;
     },
     onUpdateEngagementFailed: (data, rawEvent) => {},
+    onCallerIdMatchSucceeded: (data, rawEvent) => {
+      const { callerIdMatches } = data;
+      if (callerIdMatches.length) {
+        const firstCallerIdMatch = callerIdMatches[0];
+        if (firstCallerIdMatch.callerIdType === "CONTACT") {
+          state.incomingContactName = `${firstCallerIdMatch.firstName} ${firstCallerIdMatch.lastName}`;
+        } else if (firstCallerIdMatch.callerIdType === "COMPANY") {
+          state.incomingContactName = firstCallerIdMatch.name;
+        }
+        cti.logDebugMessage({
+          message: `Incoming call from ${state.incomingContactName} ${state.fromNumber}`,
+          type: `${callerIdMatches.length} Caller ID Matches`,
+        });
+        return;
+      }
+      cti.logDebugMessage({
+        message: `Incoming call from ${state.fromNumber}`,
+        type: "No Caller ID Matches",
+      });
+    },
+    onCallerIdMatchFailed: (data, rawEvent) => {
+      cti.logDebugMessage({
+        message: `Incoming call from ${state.fromNumber}`,
+        type: "Caller ID Match Failed",
+      });
```

Comment:
@hemang-thakkar Added an argument to log messages with a custom `type` that would be more useful for debugging.